src/main/resources/swagger.api/schemas/d2ir/returnUncirculatedDTO.json

```diff
@@ -0,0 +1,48 @@
+{
+  "description": "INN-Reach Return Circulation Message Put Request Data",
+  "type": "object",
+  "properties": {
+    "transactionTime": {
+      "description": "Transaction time",
+      "type": "integer"
+    },
+    "patronId": {
+      "description": "Patron id",
+      "type": "string",
+      "pattern": "[a-z,0-9]{1,32}"
+    },
+    "patronAgencyCode": {
+      "description": "Patron agency code",
+      "type": "string",
+      "pattern": "[a-z,0-9]{5}"
+    },
+    "itemAgencyCode": {
+      "description": "Item agency code",
+      "type": "string",
+      "pattern": "[a-z,0-9]{5}"
+    },
+    "itemId": {
+      "description": "Item id",
+      "type": "string",
+      "pattern": "[a-z,0-9]{1,32}"
+    },
```

Comment:
these fields are already defined in baseCircRequestDTO.json.
You can inherit them by adding:
`  "allOf": [
    {
      "$ref": "baseCircRequestDTO.json"
    }
  ]`

Reply:
fixed